### ISSUE TITLE
JAWS toolbar narration in  Microsoft Teams 

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#891]
Repro steps:
1. Navigate to Microsoft Teams
2. Navigate to "Type a message" compose box in Microsoft Teams. 
3. Press TAB key navigate to formatting toolbar 
4. Press left/right arrow key several times to navigate between buttons in toolbar
5. Observe JAWS narration

Actual result
When user navigate between buttons in toolbar narration is as following: 
"""
Show Formatting options (Ctrl+Shift+X) Button
To move through items press left or right arrow.
To activate press spacebar.
Emoji, GIFs and Stickers Button collapsed Has Popup dialog
To move through items press left or right arrow.
To activate press spacebar.
Rewrite with Copilot (Alt+A) Button
To move through items press left or right arrow.
To activate press spacebar.
Loop components Button
"""

Expected result
1.Instruction message "To move through items press left or right arrow. To activate press spacebar." 
should NOT be narrated every time after user navigate to button in toolbar. 
The message should be narrated only once when user navigate to toolbar. 
2.As well there is missing information that user navigated to the toolbar. I mean JAWS should narrates role of the toolbar. 

NOTE:
this bug is not reproducible with web Teams, so it looks to me it is affected by JAWS script for Microsoft Teams app

Additional Information
JAWS version and build number 
2025.2503.9

Operating System and version
Microsoft Windows 11 Enterprise
Version	10.0.22631 Build 22631

Browser and version:
Teams app client

**Comment by msftedad** [comment]
Issue is now fixed.
